apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.html.ts

```diff
@@ -321,3 +331,55 @@ function CopyCourseInstanceForm({
     </div>
   `;
 }
+
+function CourseInstanceSharing({
+  courseInstance,
+  publicLink,
+}: {
+  courseInstance: CourseInstance;
+  publicLink: string;
+}) {
+  if (!courseInstance.share_source_publicly) {
+    return html`<p>This course instance is not being shared.</p>`;
+  }
+
+  return html`
+    <p>
+      <span class="badge color-green3 me-1">Public source</span>
+      This course instance's source is publicly shared.
+    </p>
+    <div>
```

Comment:
You probably want an `mb-3` on this to space it out from the save/cancel buttons. This would match the previous spacing below the "student link" input.

Reply:
done in dd2e51b894